bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/utils/JsonUtils.java

```diff
@@ -76,6 +77,10 @@ public static Object parse(InputStream in) throws BError {
         return JsonParser.parse(in);
     }
 
+    public static Object parse(InputStream in, BTypedesc td) throws BError {
+        return JsonParser.parse(in);
+    }
+
```

Comment:
Shall we remove this because it does not do the intended work?

Reply:
This seems to be added by mistake. Will remove